lua/roslyn/config.lua

```diff
@@ -7,6 +7,8 @@ local M = {}
 ---@field config vim.lsp.ClientConfig
 ---@field choose_sln? fun(solutions: string[]): string?
 ---@field ignore_sln? fun(solution: string): boolean
+---@field choose_target? fun(targets: string[]): string?
+---@field ignore_target? fun(target: string): boolean
```

Comment:
Could you rename the types in `RoslynNvimConfig` to these new names?

`RoslynNvimConfig` is the type the user passes to the setup, and it will potentially warn the users about these fields not "existing" if they call the setup method instead of using `opts` through lazy.nvim or something like that.

Reply:
Squashed fix into 309b6d1f9200c471d6a57c4758e6c83046db68eb